client-ts/FunctionalTests/package.json

```diff
@@ -6,14 +6,25 @@
   "main": "index.js",
   "dependencies": {},
   "devDependencies": {
-    "es6-promise": "^4.2.2"
+    "@types/selenium-webdriver": "^3.0.8",
+    "@types/yargs": "^11.0.0",
+    "es6-promise": "^4.2.2",
+    "faucet": "0.0.1",
+    "selenium-webdriver": "^4.0.0-alpha.1",
+    "tap-mocha-reporter": "^3.0.6",
+    "tree-kill": "^1.2.0",
+    "ts-node": "^4.1.0",
+    "webdriver-manager": "^12.0.6",
+    "yargs": "^11.0.0"
```

Comment:
FYI @Eilon, some additional dev-time-only dependencies.

Reply:
Got it. I assume you did/will do the needful?

Reply:
Indeed I shall, just raising it FYI